lib/postcss-persistent-filter.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var postcss = require('postcss');
+var Filter = require('broccoli-persistent-filter');
+
+function PostCSS(inputNode, options) {
+  this._options = options || {};
+
+  Filter.call(this, inputNode, {
+    annotation: this._options.annotation
+  });
+
+  if (!Array.isArray(this._options.plugins)) {
+    throw new Error('postcss plugins not properly configured.');
+  }
+
+  this._processor = postcss(this._options.plugins);
+}
+
+PostCSS.prototype = Object.create(Filter.prototype);
+PostCSS.prototype.constructor = PostCSS;
+PostCSS.prototype.extensions = ['css']
+PostCSS.prototype.targetExtension = 'css';
+
+PostCSS.prototype.processString = function processString(content, relativeFile) {
+  return this._processor.process(content, {
+    from: relativeFile,
+    to: relativeFile,
+    map: {
+      inline: false,
+      annotation: false
+    }
```

Comment:
Yep, we [read that config](https://github.com/salsify/ember-css-modules/blob/master/index.js#L119-L126) and [pass it along](https://github.com/salsify/ember-css-modules/blob/master/lib/modules-preprocessor.js#L40) to `broccoli-css-modules` (as well as using it [when concatenating everything together](https://github.com/salsify/ember-css-modules/blob/master/lib/output-styles-preprocessor.js#L119))